pagopa/io-app
# chore: [IOCOM-1422,IOCOM-1425,IOCOM-1426] FIMS Consents screen

## Short description
This PR implements said screen, according to its current design.
<img width="250" alt="Screenshot 2024-06-04 at 15 37 33" src="https://github.com/pagopa/io-app/assets/60693085/2bd8a541-3818-4213-b263-21f9cee4bbe6"><img width="250" alt="Screenshot 2024-06-04 at 12 18 59" src="https://github.com/pagopa/io-app/assets/60693085/54bad3a9-3867-428c-96e3-3bf41c9e66fc">

## List of changes proposed in this pull request
- addition of required I18n keys
- various components and whatnot in order to follow the screen's design
- moved the successBody component to its own file
- minimal reducer refactor to avoid unnecessary renders
- small type refactor to avoid using deprecated functions

## How to test
While running fims' branch in the dev-server, spin up a FIMS flow and make sure that everything renders as expected.
